bitnami/valkey-cluster/templates/headless-svc.yaml

```diff
@@ -0,0 +1,28 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "common.names.fullname" . }}-headless
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- if or .Values.service.headless.annotations .Values.commonAnnotations }}
+  {{- $annotations := include "common.tplvalues.merge" ( dict "values" ( list .Values.service.headless.annotations .Values.commonAnnotations ) "context" . ) }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" $annotations "context" $) | nindent 4 }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  publishNotReadyAddresses: true
+  ports:
+    - name: tcp-redis
+      port: {{ .Values.valkey.containerPorts.valkey }}
+      targetPort: tcp-redis
+    - name: tcp-redis-bus
+      port: {{ .Values.valkey.containerPorts.bus }}
+      targetPort: tcp-redis-bus
```

Comment:
mmm, the port name in the `valkey.containerPorts` value was changed so I think we should do the same for this `tcp-redis` and `tcp-redis-bus` port names, What do you think?

Reply:
Ok no problem, I've just seen [the same in valkey](https://github.com/bitnami/charts/blob/5d703bad49025ab95654a09ecacef304de7492bd/bitnami/valkey/templates/headless-svc.yaml#L28)

Reply:
I am keeping `tcp-redis` for consistency with `valkey`chart.